### PR TITLE
USSD public - add servicerating functionality

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**
Add necessary functionality to post responses to specific questions to the correct service

### CODE REVIEW

**Comment by imsickofmaps** [comment]
@NicolaasDieBaas see example POST: https://github.com/praekelt/seed-service-rating/blob/feature/issue-1-initial-port/ratings/tests.py#L164-L172

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps please review.  of course quite a bit will change considering the requirements of the next issue (#41).

File: test/ussd_public.test.js [hunk 1]
```diff
@@ -32,6 +32,10 @@ describe("familyconnect health worker app", function() {
                             api_token: 'test_token_subscriptions',
                             url: "http://localhost:8005/api/v1/"
                         },
+                        rating: {
```
Comment (imsickofmaps) [inline test/ussd_public.test.js]: `service_rating` please

File: src/utils_project.js [hunk 2]
```diff
@@ -380,9 +380,33 @@ go.utils_project = {
...
+                    unanswered: identity.details.servicerating_unanswered,
+                    invite_uuid: identity.details.invite,
+                };
             });
     },
 
+    // saves servicerating info
+    post_servicerating_feedback: function(im, q_id, q_text, answer_text, answer_value, version_number, invite_uuid) {
+        var payload = {
+            "identity_id": im.user.answers.user_id,
```
Comment (imsickofmaps) [inline src/utils_project.js]: Just `identity` (fixture too)

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps changes good to merge now?

File: src/utils_project.js [hunk 3]
```diff
@@ -400,7 +400,7 @@ go.utils_project = {
         };
 
         return go.utils
-            .service_api_call("rating", "post", null, payload, "rating/", im)
+            .service_api_call("service_rating", "post", null, payload, "service_rating/", im)
```
Comment (imsickofmaps) [inline src/utils_project.js]: just the name of the service in the config needed to change, the second variable still needs to point to the `rating` app.

**Comment by imsickofmaps** [comment]
@NicolaasDieBaas one more change, last was too eager

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps now? ;)

**Comment by imsickofmaps** [comment]
:+1:

**Pull request merged**